apps/et/et-cron-wa-task-for-expired-bfactions/demo-00.yaml

```
@@ -1,9 +1,9 @@
 apiVersion: helm.toolkit.fluxcd.io/v2
 kind: HelmRelease
 metadata:
-  name: et-cron-wa-task-for-expired-bfactions
+  name: demo-et-cron-wa-task-for-expired-bfactions
 spec:
-  releaseName: et-cron-wa-task-for-expired-bfactions
+  releaseName: demo-et-cron-wa-task-for-expired-bfactions
```

These can stay how they are
```suggestion
  name: et-cron-wa-task-for-expired-bfactions
spec:
  releaseName: et-cron-wa-task-for-expired-bfactions
```